CHANGELOG.md

```diff
@@ -1,8 +1,52 @@
+### 3.40.0
+
+- In the previous versions the following code will throw an error:
```

Comment:
Thanks chatgpt for that : 

> Is it better to say : 
> 
>  In the previous versions
> 
> or 
> 
>  In previous versions

Response :

> Both phrases are grammatically correct, but the choice between them depends on the context and the specific message you want to convey.
> 
> "In the previous versions": Use this phrase when you want to emphasize a specific set of versions or a particular instance of previous versions. It suggests that you are referring to a specific group of versions that you have mentioned or will mention in the conversation.
> Example: "In the previous versions of the software, there were several bugs that we needed to fix."
> 
> "In previous versions": Use this phrase when you are discussing a general characteristic or trend that applies to multiple versions without specifying which ones. It implies that the information about previous versions is relevant in a broader sense.
> Example: "In previous versions, the user interface was less intuitive."
> 
> In summary, choose "In the previous versions" when you want to be specific about which versions you are referring to, and use "In previous versions" when discussing a general trend or characteristic that applies to multiple versions without specifying them individually.

Reply:
Since all versions before that were affected, I think it should be "in previous versions"